src/bors/event.rs

```diff
@@ -18,6 +18,8 @@ pub enum BorsRepositoryEvent {
     /// A check suite has been completed, either as a workflow run on Github Actions, or as a
     /// workflow from some external CI system.
     CheckSuiteCompleted(CheckSuiteCompleted),
+    /// When a repository's tree state changes (open/closed)
+    TreeStateChanged(TreeStateChanged),
```

Comment:
I don't think that we need any event for this? It should be enough to just handle the comment and store information about closed/opened repo in the DB.